### ISSUE TITLE
Somewhat scary UI for the device code web interface

### ISSUE THREAD

**Issue by bbockelm** [indigo-iam/iam#300]
When showing the device code flow to someone, they remarked the landing page provides very little context to what is happening:

![image](https://user-images.githubusercontent.com/1093447/65889621-2d244880-e367-11e9-8bd0-f567a96f1ebf.png)

Is it possible to include a sentence or two explaining what the user is looking at?

**Comment by andreaceccanti** [comment]
yeah, that's the default device code approval shown by MitreID Connect that I also find suboptimal from an UI point of view, and we have in the backlog to improve it.

Do you have suggestions on what kind of information you'd want to see in this page?

**Comment by andreaceccanti** [comment]
btw: the context is mostly known "after" having entered the code

**Comment by bbockelm** [comment]
How about something like this:

Enter Code
========
This is the first step to authorize a client to act on your behalf; on the next page, you will be given the opportunity to approve or deny the client.  The client that sent you to this page should have also provided you with a short code; copy it into the text box below and click "submit".  No permissions will be granted to the client until final confirmation is given on the next page.
